src/cards/migrations/0010_auto_20180330_0942.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2018-03-30 09:42
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def migrate_help_links(apps, schema_editor):
+    Cards = apps.get_model("cards", "Card")
+    for card in Cards.objects.all():
+        new_value = card.help_links
+        if '\n' in new_value:
```

Comment:
`new_values.splitlines()` should work. No?

Reply:
Didn't know about this method and you are right!